netdata-installer.sh

```diff
@@ -1022,6 +1027,171 @@ install_go() {
 
 install_go
 
+function get_kernel_version() {
+  r="$(uname -r | cut -f 1 -d '-')"
+
+  read -r -a p <<< "$(echo "${r}" | tr '.' ' ')"
+
+  printf "%03d%03d%03d" "${p[0]}" "${p[1]}" "${p[2]}"
+}
+
+detect_libc() {
+  libc=
+  if ldd --version 2>&1 | grep -q -i glibc; then
+    echo >&2 " Detected GLIBC"
+    libc="glibc"
+  elif ldd --version 2>&1 | grep -q -i 'gnu libc'; then
+    echo >&2 " Detected GLIBC"
+    libc="glibc"
+  elif ldd --version 2>&1 | grep -q -i musl; then
+    echo >&2 " Detected musl"
+    libc="musl"
+  else
+    echo >&2 " ERROR: Cannot detect a valid libc on your system!"
```

Comment:
I think this should probably be 'a supported libc' not 'a valid libc'. It's Linux, there's pretty much always going to be _some_ libc there, we just don't support anything other than musl and glibc 9and technically eglibc by virtue of supporting glibc).

Reply:
Fine :) Bust some Liniux(es) don't ship a libc a all 🤣